node/node.go

```diff
@@ -79,6 +80,10 @@ type Node struct {
 
 	mu            sync.Mutex
 	CurrentSocket string
+
+	// BlobVersionParams is a map of blob version parameters loaded from the chain.
+	// It is used to determine blob parameters based on the version number.
+	BlobVersionParams *corev2.BlobVersionParameterMap
```

Comment:
This needs mutex protection for concurrent read/write

Reply:
The underlying map is read only and this pointer to the read only map gets swapped out atomically occasionally. Do we need synchronization here?

Reply:
I think you're right. Converted these into `atomic.Pointer`